### PR TITLE
Require Node.js 12+ to execute the creator

### PR DESCRIPTION
**Pull request by AlCalzone**
**PR Checklist:**

-   [x] Provide a meaningful description to this PR or mention which issues this fixes.
-   [ ] ~~Add tests for your change. This includes negative tests (i.e. inputs that need to fail) as well as baseline tests (i.e. how should the directory structure look like?).~~
-   [ ] ~~Run the test suite with `npm test`~~ No functional changes
-   [ ] ~~If there are baseline changes, review them and make a separate commit for them with the comment "accept baselines" if they are desired changes~~
-   [ ] ~~Ensure the project builds with `npm run build`~~ No functional changes
-   [ ] ~~If you added a required option, also add it to the template creation (`.github/create_templates.ts`)~~
-   [ ] ~~Add a detailed migration description to `docs/updates` explaining what the user needs to do when manually updating an existing project~~ Nothing to do
-   [x] Add your changes to `CHANGELOG.md` (referencing the migration description and this PR or the issue you fixed)

**Description:**  
Since more and more dependencies require Node.js 12, we need to stay up to date. This PR requires Node 12 to run the creator.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Command line utility to quickly create a new adapter or VIS widget for [ioBroker
 
 ## Prerequisites
 
-Any computer with NodeJS in version 8 or higher and npm 6 or higher.
+Any computer with NodeJS in version 12 or higher and npm 6 or higher.
```
Comment (UncleSamSwiss) [inline README.md]: We should not implicitly suggest that npm 7 is supported. I think, there should be an explicit mention that we currently *do not* support npm 7.

Comment (AlCalzone) [reply README.md]: Uhm why should the creator not be working on npm7?

Comment (Apollon77) [reply README.md]: Agree, npm7 is currently only an issue when installing adapters or stuff from GitHub

Comment (AlCalzone) [reply README.md]: I've added a note.

**Review by UncleSamSwiss — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**